Tutorial: A non-technical guide to understanding data in PostHog (#4768)
PostHog/posthog.com: contents/tutorials/non-technical-guide-to-data.md

---
title: A non-technical guide to understanding data in PostHog
sidebar: Docs
showTitle: true
author: ['ian-vanagas']
date: 2022-11-28
featuredImage: ../images/tutorials/banners/non-technical-guide-to-data.png
topics: ["data management"]
---

> Non-technical users have many tools in PostHog for gaining insights from their product data. You don’t need coding knowledge, but you do need some knowledge of your data. Knowing the formatting and structure of your data is key to getting the most out of PostHog as a non-technical user.

Reviewer: ```suggestion
Non-technical users have many tools in PostHog for gaining insights from their product data. You don’t need to be a software engineer, but you do need some knowledge of your data. Knowing the formatting and structure of your data, for example, is key to getting the most out of PostHog as a non-technical user.
```
Just a minor nit to soften this up a bit.

In this tutorial, we’ll go through how you can understand your product’s data in PostHog. This allows you to find the data you want, format it in the right way, and generate insights to improve your product. 

> This tutorial assumes your technical team set up PostHog in your app or site, and you have data to work with. Read our [event tracking guide](/tutorials/event-tracking-guide) for details on setting this up if not.

## 1. Explore other’s creations

> In school, using someone else’s work is cheating, but in real life, it is “collaboration.” Exploring teammates’ work in PostHog is a great way to get started understanding the data there.

Reviewer 2: ```suggestion
In school, using someone else’s work is cheating; in real life, it's “collaboration.” Exploring teammates’ work in PostHog is a great way to get started understanding the data there.
```

This made me lol. I'd suggest engaging the awesome power of the semi-colon to make it hit even better. Semi-colons are gangster.

> The best place to do this is though the dashboards and insights other people in your organization created. To find them, click “Dashboards” on the sidebar. If you’re looking for the most important ones, they might be under “Pinned.”

Reviewer 2: ```suggestion
The best place to do this is through the dashboards and insights other people in your organization created. To find them, click “Dashboards” on the sidebar. If you’re looking for the most important ones, they might be under “Pinned.”
```

![Dashboards](../images/tutorials/non-technical-guide-to-data/dashboards.png)

> When you click into a dashboard, don’t only look at the visualizations, but click “show details” to show the queries and filters behind them. Even better, click on the title of the visualizations, go to the insight detail view, and click Edit (don’t worry, we won’t change anything).

Reviewer: ```suggestion
When you click into a dashboard, don’t only look at the visualizations, but click “show details” to show the queries and filters behind them. Even better, click on the title of the visualizations, go to the insight detail view, and click Edit (don’t worry, we won’t save the changes).
```

When finished, head back to the dashboards and check out another insight. Repeating this will start to build familiarity with the types of visualizations PostHog has, and how your data fits into them.

## 2. See all the types of data

Once you’ve explored dashboards and understood how data is used, it is time to dive deeper into the different types. You can see all of your organization’s data types under the “Data management” tab. In this section, you can see all the captured event types, created actions, and event properties.

You can sort all of these types of data to learn how often your team is using it. For example, by sorting by “30-day volume,” you can see the number of those events your product captures each month. By sorting by “30-day query count,” you can see how often your team uses events or properties to calculate visualizations (insights) in PostHog.

![Data management](../images/tutorials/non-technical-guide-to-data/data-management.png)

Understanding the events, actions, and properties in PostHog, and what your team is actually using, should give you some ideas of what data is available, and what is important. This is useful when you go to create insights.

> ## 3. See you captured events

Reviewer: ```suggestion
## 3. See your captured events
```

> Another way to understand the data in PostHog is to see the captured events from your usage. This helps you connect data in PostHog with product usage in the real world. You’ll be able to relive your usage of the product through the data inside PostHog

Reviewer: This is an A+ addition to this article IMHO
